src/main/client_context.cpp

```diff
@@ -319,6 +319,8 @@ shared_ptr<PreparedStatementData> ClientContext::CreatePreparedStatement(ClientC
 			planner.parameter_data.emplace_back(value);
 		}
 	}
+
+	client_data->http_state = make_uniq<HTTPState>();
```

Comment:
Looking elsewhere, it seems it should probably be `make_shared<HTTPState>`?

And also I am not sure whether either an assertion or a test on http_state being already NOT initialised would not make sense (but then it's missing also elsewhere, so not really tied to this PR).